kibble/api/tv_test.go

```diff
@@ -242,8 +242,9 @@ func TestSeasonClassifications(t *testing.T) {
 
 	assert.Equal(t, 2, len(model.Classifications))
 
-	assert.Equal(t, "au", model.Classifications[0].CountryCode)
-	assert.Equal(t, "nz", model.Classifications[1].CountryCode)
+	countryCodes := [2]string{model.Classifications[0].CountryCode, model.Classifications[1].CountryCode}
```

Comment:
Nice catch.

Wouldn't be [assert.ElementsMatch](https://github.com/stretchr/testify/blob/083ff1c0449867d0d8d456483ee5fab8e0c0e1e6/assert/assertions.go#L894) a better option here?

```
	want := []models.Classification{
		models.Classification{
			CountryCode: "au",
			Label:       "Australian Label",
			Description: "Australian Description",
		},
		models.Classification{
			CountryCode: "nz",
			Label:       "NZ Label",
			Description: "NZ Description",
		},
	}

	assert.ElementsMatch(t, model.Classifications, want)
```